velero-plugin-for-microsoft-azure/volume_snapshotter.go

```diff
@@ -413,12 +425,22 @@ func (b *VolumeSnapshotter) SetVolumeID(unstructuredPV runtime.Unstructured, vol
 		return nil, errors.WithStack(err)
 	}
 
-	if pv.Spec.AzureDisk == nil {
-		return nil, errors.New("spec.azureDisk not found")
-	}
+	if pv.Spec.CSI != nil {
+		if pv.Spec.CSI.Driver == diskCSIDriver {
+			pv.Spec.CSI.VolumeHandle = getComputeResourceName(b.disksSubscription, b.disksResourceGroup, disksResource, volumeID)
+			if _, exist := pv.Spec.CSI.VolumeAttributes["csi.storage.k8s.io/pv/name"]; exist {
+				pv.Spec.CSI.VolumeAttributes["csi.storage.k8s.io/pv/name"] = volumeID
```

Comment:
I don't find too much description about the attribute `csi.storage.k8s.io/pv/name` but it seems the name of the PV not the volume ID of the cloud provider?